cardea-server/src/main/java/ca/on/oicr/gsi/cardea/server/service/CaseService.java

```diff
@@ -178,18 +178,25 @@ private ShesmuSequencing makeShesmuSequencing(List<Sample> samples, MetricCatego
     boolean hasWaiting = false;
     for (Sample sample : samples) {
 
-      if (sample.getDataReviewPassed() == null || sample.getQcPassed() == null) {
+      if (sample.getDataReviewPassed() == null || (sample.getQcPassed() == null && sample.getQcUser() == null)) {
```

Comment:
Doesn't hurt, but there's no need to check `qcPassed`. If `qcUser` is null, then QC is pending. Same for run QC below